src/results/electric_storage.jl

```diff
@@ -23,7 +23,10 @@ function add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::
     r = Dict{String, Any}()
     r["size_kwh"] = round(value(m[Symbol("dvStorageEnergy"*_n)][b]), digits=2)
     r["size_kw"] = round(value(m[Symbol("dvStoragePower"*_n)][b]), digits=2)
-
+    if b in p.s.storage.types.dc_coupled
+        r["dc_coupled_inverter_size_kw"] = round(value(m[:dvDCCoupledTechStorageInverterSizeDC][b]), digits=2)
```

Comment:
from what little I've seen, it seems like these inverters are commonly rated by their AC output, but these are kWDC units, right? Would it make sense to convert to AC sizing? If not, I think we should at least specify the units in the name since they differ from the AC units of the standard `size_kW`

Reply:
Oh thanks! I wasn't sure whether the inverter should be defined by the ac or dc side so I just picked one and I meant to come back to that decision. I will change to dvDCCoupledTechStorageInverterSizeAC.

Reply:
done in commit 6214f80a